cloudwatch_agent_endpoint_sg.tf

```diff
@@ -0,0 +1,24 @@
+# Security group for the interface endpoints used by the CloudWatch
+# agent in the private subnet
+resource "aws_security_group" "cloudwatch_agent_endpoint" {
+  provider = aws.sharedservicesprovisionaccount
+
+  vpc_id = aws_vpc.the_vpc.id
+
+  tags = {
+    Name = "CloudWatch agent endpoints"
```

Comment:
Is there a reason the resource name is singular but the tag name is plural?

Reply:
The security group is applied to multiple VPC endpoints, since access to multiple VPC endpoints is required to make use of the CloudWatch Agent.